auv_sim/auv_sim_bringup/launch/start_gazebo.launch

```diff
@@ -19,12 +18,10 @@
   <arg name="yaw" value="-1.8" />
 
   <include file="$(find auv_sim_bringup)/launch/inc/start_world.launch.xml">
-    <arg name="gui" default="$(arg use_gui)" />
-    <arg name="debug" default="$(arg debug)" />
-    <arg name="world_name" default="$(arg world_name)" />
-    <arg name="namespace" default="$(arg namespace)" />
-    <arg name="velocity_control" default="$(arg velocity_control)" />
-    <arg name="joy_id" default="$(arg joy_id)" />
+    <arg name="gui" value="$(arg use_gui)" />
+    <arg name="debug" value="$(arg debug)" />
+    <arg name="world_name" value="$(arg world_name)" />
+    <arg name="namespace" value="$(arg namespace)" />
```

Comment:
Why set default something that is already set default up top?
Switched default to value. Keeps things consistent and avoids duplication.